src/handler.ts

```diff
@@ -0,0 +1,13 @@
+import { PlatformServerless } from '@tsed/platform-serverless';
+
+import { V2_CONTROLLERS, V3_CONTROLLERS } from './controllers';
+// import { swaggerConfig } from './config/constants';
```

Comment:
we dont support swagger anymore?

Reply:
we will need to re enable swagger or figure out how it fits in the new platform. we will not release up directly until yeah we support it again.